languages/en-US/administration/configuration.json

```
@@ -2,34 +2,34 @@
     "DESCRIPTION": "Shows the server configuration!",
     "USAGE": "{{prefix}}configuration",
     "EXAMPLES": "{{prefix}}configuration",
-    "PREFIX_TITLE": "Server prefix",
-    "IGNORED_CHANNELS_TITLE": "Ignored channel(s)",
+    "PREFIX_TITLE": "\n- Server prefix:",
+    "IGNORED_CHANNELS_TITLE": "- Ignored channel(s):",
     "NO_IGNORED_CHANNELS": "No ignored channels.",
-    "AUTOROLE_TITLE": "Autorole",
+    "AUTOROLE_TITLE": "- Autorole:",
```

Sans le tiret c'est plus propre visuellement 
```suggestion
    "AUTOROLE_TITLE": "Autorole:",
```